src/WebJobs.Script/Description/DotNet/FunctionAssemblyLoadContext.cs

```diff
@@ -84,7 +86,10 @@ internal static (IDictionary<string, RuntimeAsset[]> depsAssemblies, IDictionary
                 {
                 }
             }
-
+            else
+            {
```

Comment:
Reproes locally ! Without `functions.deps.json`  - Running V2 app using V3 Core tools throws - Platform not supported exception